homeassistant/components/cover/strings.json

```diff
@@ -1,5 +1,17 @@
 {
   "device_automation": {
+    "action_type": {
+      "open": "Open {entity_name}",
+      "close": "Close {entity_name}",
+      "toggle": "Toggle {entity_name}",
```

Comment:
Since we can't come up with a good name, let's remove it for now and let's not keep it blocking this PR. I would like to see this merged before the next beta.

Reply:
Removed!